Index: perftest-3.4/configure.ac
===================================================================
--- perftest-3.4.orig/configure.ac
+++ perftest-3.4/configure.ac
@@ -271,7 +271,7 @@ if [test $HAVE_PACKET_PACING = yes]; the
 	AC_DEFINE([HAVE_PACKET_PACING], [1], [Have PACKET_PACING support])
 fi
 
-CFLAGS="-g -Wall -D_GNU_SOURCE -O3"
+CFLAGS+=" -g -Wall -D_GNU_SOURCE -O3"
 AC_SUBST([LIBUMAD])
 AC_SUBST([LIBMATH])
 AC_CONFIG_FILES([Makefile])
